roles/nova-data/templates/etc/nova/nova.cinder_encryption.conf

```diff
@@ -1,2 +1,11 @@
+{% if cinder.encryption.fixed_key is defined %}
 [keymgr]
```

Comment:
Please use `[KEY_MANAGER]` instead, [keymgr](https://docs.openstack.org/newton/config-reference/block-storage/samples/cinder.conf.html) is deprecated